openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -471,6 +471,7 @@ def get_schedules(self):
         schedules = Schedule.objects.select_related('enrollment').filter(
             self.experience_filter,
             active=True,
+            enrollment__is_active=True,
```

Comment:
Keeping this line to account for old schedules that would not have been able to use the new signal handler. In a couple months, this should be safe to remove, but it's also fine to leave since it won't affect the query numbers.